src/server/qgsserverapiutils.h

```diff
@@ -225,8 +225,10 @@ class SERVER_EXPORT QgsServerApiUtils
     /**
      * Returns a \a crs as OGC URI (format: http://www.opengis.net/def/crs/OGC/1.3/CRS84)
      * Returns an empty string on failure.
+     *
+     * \deprecated since QGIS 3.30 use QgsCoordinateReferenceSystem::toOgcUri() instead
      */
-    static QString crsToOgcUri( const QgsCoordinateReferenceSystem &crs );
+    Q_DECL_DEPRECATED static QString crsToOgcUri( const QgsCoordinateReferenceSystem &crs ) SIP_DEPRECATED;
```

Comment:
Undeprecate in the backport?

Reply:
Thanks I un-deprecated it

Reply:
@m-kuhn I hadn't rights to push it here so I opened a new pr https://github.com/qgis/QGIS/pull/50745. Could you close this one please?